scheduling/round_robin.py

```diff
@@ -4,8 +4,76 @@
 https://en.wikipedia.org/wiki/Round-robin_scheduling
 """
 from __future__ import annotations
-
 from statistics import mean
+import time
+from itertools import cycle as cy
+from shutil import get_terminal_size as gts
+from threading import Thread as th

```

Comment:
Please do not replace standard names with cryptic names.
```suggestion
from itertools import cycle
from shutil import get_terminal_size
from threading import Thread

```